Content.Server/SimpleStation14/EndOfRoundStats/ShotsFired/ShotsFiredStatSystem.cs

```diff
@@ -0,0 +1,50 @@
+using Content.Server.GameTicking;
+using Content.Shared.GameTicking;
+using Content.Shared.Weapons.Ranged.Components;
+using Content.Shared.Weapons.Ranged.Events;
+
+namespace Content.Server.SimpleStation14.EndOfRoundStats.ShotsFired;
+
+public sealed class ShotsFiredStatSystem : EntitySystem
+{
+    int shotsFired = 0;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<GunComponent, AmmoShotEvent>(OnShotFired);
+
+        SubscribeLocalEvent<RoundEndTextAppendEvent>(OnRoundEnd);
+        SubscribeLocalEvent<RoundRestartCleanupEvent>(OnRoundRestart);
+    }
+
+    private void OnShotFired(EntityUid _, GunComponent __, AmmoShotEvent args)
+    {
+        shotsFired++;
+    }
+    private void OnRoundEnd(RoundEndTextAppendEvent ev)
+    {
+        var line = "\n[color=cadetblue]";
+
+        if (shotsFired < 25 && shotsFired != 0)
```

Comment:
```suggestion
        if (shotsFired < 25)
```

Reply:
Huh? Why?

Reply:
< 25 would cover the !0

Reply:
If it's less than 25 AND doesn't equal zero. So, if it's 1 - 24, or below zero.